Dependencies/napi/source/js_native_api_JavaScriptCore.cc

```diff
@@ -205,7 +202,8 @@ namespace {
       JSObjectSetPrototype(env->context, constructor, prototype);
       
       JSValueRef exception{};
-      JSObjectSetProperty(env->context, prototype, JSString("constructor"), constructor, kJSPropertyAttributeReadOnly | kJSPropertyAttributeDontDelete, &exception);
+      JSObjectSetProperty(env->context, prototype, JSString("constructor"), constructor,
```

Comment:
Nit: Is this line break intentional? I thought it was perfectly readable on one line, so wasn't sure if this was some automatic tooling.

Reply:
I did this manually in this case. I try to keep NAPI conventions which seems to limit the line length.